engine/src/main/scala/org/scalarules/dsl/nl/grammar/DslNumberLikeListAggregator.scala

```diff
@@ -1,31 +0,0 @@
-package org.scalarules.dsl.nl.grammar
-
-import org.scalarules.finance.core.NumberLike
-import org.scalarules.engine.{Derivation, Evaluation, Fact}
-
-trait DslNumberLikeListAggregator {
```

Comment:
Was it not possible to make Git see this file as renamed? Seeing it as deleted might complicate merging, for example with Translation branch